src/Microsoft.AspNetCore.Razor.Evolution/Intermediate/HtmlContentIRNodeExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Text;
+
+namespace Microsoft.AspNetCore.Razor.Evolution.Intermediate
+{
+    internal static class HtmlContentIRNodeExtensions
+    {
+        private static StringBuilder _builder;
```

Comment:
Can use this code https://github.com/dotnet/coreclr/blob/master/src/mscorlib/src/System/Text/StringBuilderCache.cs